doc/programming_guide/extending.rst

```diff
@@ -41,6 +41,8 @@ The following formula was used to convert CPU% to MIPS:
 
 MIPS = (CPU% / 100%) * (600 MHz / 5 cores)
```

Comment:
This is in existing documentation, but would love for this to be a slightly fancier equation formatting!